src/main/scala/org/broadinstitute/dsde/workbench/sam/service/UserService.scala

```diff
@@ -223,8 +224,9 @@ object UserService {
   def genWorkbenchUserId(currentMilli: Long): WorkbenchUserId =
     WorkbenchUserId(genRandom(currentMilli))
 
-  def validateEmailAddress(email: WorkbenchEmail): IO[Unit] = {
+  def validateEmailAddress(email: WorkbenchEmail, domainBlacklist: Seq[String]): IO[Unit] = {
```

Comment:
Is there value in passing the `domainBlacklist` around as an argument to this method?  As far as I understand, the field `UserService.domainBlacklist`will not change on they fly, and so I don't see the value in passing it in here every time it is called.

Reply:
it is because this is a static function, it does not have access to config